### ISSUE TITLE
NVDA with google crome and firefox

### ISSUE THREAD

**Issue by BEdenhofer** [nvaccess/nvda#8036]
When typing in the address bar with google chrome or firefox, NVDA anounces everything you type. for example, it says www selection removed, and it keeps repeating as auto complete! when you type in the address bar, it should only anounce one letter, at a time, like microsoft edge for example when you type in the address bar with microsoft edge, it says www.nvaccess.org and it doesn't fill it in as auto complete it only says one letter at a time.

**Comment by leonardder** [comment]
This is browser specific behaviour. As for Firefox, I belief you can disable address bar auto completion, I"m not sure about Chrome. Anyway, unless we believe that this should be fixed on nVDA's end, which I wouldn't, I'm inclined to close this as a won't fix.

**Comment by zahra21** [comment]
hi.
type about:config in address bar of firefox.
press enter to open experts configurations and accept the risk after recieving the worning.
then in the search bar, type or peist
autocomplete
and press enter to make:
browser.urlbar.autocomplete.enabled false.
this is a problem of firefox 50 and later, not nvda bug!
hope that help.

**Comment by ehollig** [comment]
I am closing this as browser specific behavior.
